### ISSUE TITLE
API accessability HTTPS + custom domain

### ISSUE THREAD

**Issue by ZakBrinlee** [ActoKids/AD440_W19_CloudPracticum#33]
API needs to be accessible from HTTPS from a custom domain.

Estimated time:

Actual time:

**Comment by MrSamsa42** [comment]
Estimated time: 5 hours....
Actual time:  8+ (because it takes time for changes to deploy)

I successfully added HTTPS to my own domain and set it up to re-route HTTP requests to HTTPS.  So, http://corpsitron.com will re-route to a secure site.  

However, I have so far not been been able to map the ping api to my custom url.  So, https://api.corpsitron.com/v1 isn't working.

**Comment by MrSamsa42** [comment]
UPDATE:  ping api is working -- I just had the path wrong.  Correct path is https://api.corpsitron.com/v1/ad440TestFunction

Ideally, path should be https://api.corpsitron.com/v1/ping
I will make the necessary corrections

**Comment by MrSamsa42** [comment]
Estimated time:  5 hours

Actual time: 10+

It took a very long time to determine the correct sequence of steps to get everything set up correctly.   This was made worse by the fact that it can take 30+ minutes for changes to deploy.  

wiki: https://github.com/ActoKids/api/wiki/API-accessability-HTTPS---custom-domain

Working PING API from custom domain with valid certificate: https://api.jradler.com/v1/ping

Link to PR: https://github.com/ActoKids/api/pull/23

**Comment by toddysm** [comment]
Good.

Need few more things for this sprint:
- Who tested this and what is her/his feedback?
- What did you test and what is your feedback?

**Comment by MrSamsa42** [comment]
> Good.
> 
> Need few more things for this sprint:
> 
> * Who tested this and what is her/his feedback?
  It was tested by Alex.  His feedback is here: ActoKids/api#23

> * What did you test and what is your feedback?
I reviewed Alex's swagger file.  I copied the file into the Swagger editor, and the editor did not flag any errors.  However, API Gateway console rejects the file when I try to import it, citing multiple errors related to "Unsupported keywords".  I made comments and suggestions related to the content of the file.  Those were made in-line and can be shown by clicking "Show conversation" here
https://github.com/ActoKids/api/pull/21/files/ef86d4184d78930fed2a32fdb66c5cda42f62123
